lib/Cake/View/Elements/Flash/default.ctp

```diff
@@ -1 +1,7 @@
-<div id="<?php echo $key; ?>Message" class="<?php echo !empty($params['class']) ? $params['class'] : 'message'; ?>"><?php echo $message; ?></div>
\ No newline at end of file
+<?php
+$class = 'message';
+if (!empty($params['class'])) {
+    $class .= ' ' . $params['class'];
+}
+?>
+<div class="<?= h($class) ?>"><?= h($message) ?></div>
```

Comment:
The lack of id attribute may cause applications to be styled incorrectly when they upgrade. Can we retain the id attribute for now?